linode/manager
# Rename NR script to invalidate cache

## Description

Rename the NR script so the the previous script isn't loaded from cache by the browser.

**To test:**
Remove L10 and L12 of `index.html` and load the app. `new-relic.js` should be loaded,
